Makefile

```diff
@@ -21,36 +24,72 @@ DIR_PROBE := "$(CUR_DIR)/cmd/probe"
 build: build-runner build-probe
 
 build-runner: deps-runner
-	@cd ${DIR_RUNNER} && go build -ldflags="-X 'github.com/grafana/nethax/pkg/kubernetes.ProbeImageVersion=${PROBE_VERSION}'" -o "$(CUR_DIR)/bin"
+	@cd $(DIR_RUNNER) && go build -ldflags="-X 'github.com/grafana/nethax/pkg/kubernetes.ProbeImageVersion=$(PROBE_VERSION)'" -o "$(CUR_DIR)/bin"
 
 build-probe: deps-probe
-	@cd ${DIR_PROBE} && go build -ldflags="-X 'github.com/grafana/nethax/pkg/kubernetes.ProbeImageVersion=${PROBE_VERSION}'" -o "$(CUR_DIR)/bin"
+	@cd $(DIR_PROBE) && go build -ldflags="-X 'github.com/grafana/nethax/pkg/kubernetes.ProbeImageVersion=$(PROBE_VERSION)'" -o "$(CUR_DIR)/bin"
 
+.PHONY: clean
 clean:
 	@rm -f "$(CUR_DIR)/bin/"*
 
+.PHONY: deps deps-runner deps-probe
+
+# Default kind cluster name if not overridden
+KIND_CLUSTER_NAME ?= "nethax"
+
 deps: deps-runner deps-probe
 
 deps-runner:
-	@cd ${DIR_RUNNER} && go mod download
+	@cd $(DIR_RUNNER) && go mod download
```

Comment:
Given that we only have one `go.mod` file at the root of the repo, executing `go mod download` there will download both the dependencies for `cmd/runner` and `cmd/probe`, no need to run two separated commands.

Reply:
lol that's a good point 😅